activation-config.yaml

```diff
@@ -56,6 +56,10 @@ googleCloud:
     # Send custom metrics to GCP to diagnose quickly if your dynatrace-gcp-function processes and sends metrics to Dynatrace properly.
     # Allowed values: true/yes, false/no
     selfMonitoringEnabled: false
+    # scopingProjectSupportEnabled: set to true when metrics scope is configured. Metrics will be collected from all monitored projects,
+    # but topology fetch will not, so we need to bypass the verification in case there are instances of a service in monitored projects
+    # and not in the scoping project.
```

Comment:
What happens if you have metric scope for other project but this variable is set to false by mistake?

Reply:
This is uncertain, because if you have instances of a service in the scoping project, you will collect metrics from the other monitored projects as well, because it will pass the topology search. I don't know if there's anything to be done in that case, to me it is a misconfiguration, because you set the GCP env/main project to collect metrics from other projects, but you put false here.